src/EditorFeatures/CSharpTest/RawStringLiteral/RawStringLiteralCommandHandlerTests.cs

```diff
@@ -120,6 +118,105 @@ public void TestReturnInSixQuotes_Interpolated()
     """"""");
         }
 
+        [WpfFact]
+        public void TestReturnInSixQuotesMoreQuotesLaterOn()
+        {
+            using var testState = RawStringLiteralTestState.CreateTestState(
```

Comment:
Can you explain what the previous vs current behavior is?

Reply:
previous behavior was that we didn't even support this because we accidentally counted the later quotes as well as the immediately following quotes.  So you got no special behavior.